frontend/src/App.js

```diff
@@ -1,30 +1,18 @@
-import React from 'react';
-import logo from './logo.svg';
+// Import libraries
+import React, { Component } from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// Import local files
```

Comment:
Removed the App.js boilerplate code and added in a top level App component. App Component is the main component in React which acts as a container for all other components.